PointsRemovalScripts/HPR_open3d.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+import open3d as o3d
+
+def hidden_points_removal(path):
+    str(path)
+    pcd = o3d.io.read_point_cloud(path)
+    diameter = np.linalg.norm(
+        np.asarray(pcd.get_max_bound()) - np.asarray(pcd.get_min_bound()))
+    #o3d.visualization.draw_geometries([pcd])
+
+    #Define parameters used for hidden_point_removal
+    camera = [0, 0, 0]
+    radius = diameter * 100
+
+    #Get all points that are visible from given view point
```

Comment:
```suggestion
    # Get all points that are visible from given view point
```
Let's fix it everywhere